typescript/create-onchain-agent/templates/actionProvider/actionProvider.njk

```diff
@@ -0,0 +1,20 @@
+import { customActionProvider{% if walletProvider !== "none" %}, {{ walletProvider }}{% endif %} } from "@coinbase/agentkit";
+import { {{ schemaName }} } from "./schemas";
+import { z } from "zod";
+
+/**
+ * Creates a {{ className }} action provider.
+ */
+export const {{ name }} = () => customActionProvider{% if walletProvider !== "none" %}<{{ walletProvider }}>{% endif %}({
+  name: "{{ actionName }}",
+  description: `This tool will perform a {{ className }} operation.`,
+  schema: {{ schemaName }},
+  invoke: async ({% if walletProvider !== "none" %}wallet: {{ walletProvider }}, {% endif %}args: z.infer<typeof {{ schemaName }}>) => {
+    try {
+      // Do work here
+      return `Successfully performed {{ actionName }} and returned the response`;
+    } catch (error) {
+      return `Error performing {{ actionName }}: Error: ${error}`;
+    }
+  },
+}); 
```

Comment:
this formatting is off in the generated code, it should be like:
```ts
export const myactionActionProvider = () =>
  customActionProvider<EvmWalletProvider>({
    name: "myaction_action",
    description: `This tool will perform a MyactionActionProvider operation.`,
    schema: MyactionActionSchema,
    invoke: async (wallet: EvmWalletProvider, args: z.infer<typeof MyactionActionSchema>) => {
      try {
        // Do work here
        return `Successfully performed myaction_action and returned the response`;
      } catch (error) {
        return `Error performing myaction_action: Error: ${error}`;
      }
    },
  });
```

let's also add a blank newline to the end of the file